### PR TITLE
Migrated TC test_256_snapshots

### PR DESCRIPTION
**Pull request by nik-redhat**
Migrated TC [test_256_snapshots](https://github.com/gluster/glusto-tests/blob/master/tests/functional/snapshot/test_256_snapshots.py)

Updates: #292 
Signed-off-by: nik-redhat <nladha@redhat.com>

### CODE REVIEW

**Comment by nik-redhat** [comment]
@srijan-sivakumar for this TC I doubt we will have the hardware to create 256 snapshots? Shall I add it to the excluded list?

**Comment by srijan-sivakumar** [comment]
For this TC, we would need a baremetal server or a VM with good amount of resource. Both of which are not with us currently. I think this again needs to be put into the exclude list with the proper description/reasoning.

**Comment by nik-redhat** [comment]
> For this TC, we would need a baremetal server or a VM with good amount of resource. Both of which are not with us currently. I think this again needs to be put into the exclude list with the proper description/reasoning.

Yeah, made the update.

File: config/config.yml [hunk 1]
```diff
@@ -66,3 +66,4 @@ excluded_tests:
     - tests/functional/afr/test_arb_to_repl_conversion_with_io.py # Heal not completing even after increasing the timeout and the IO is completed
     - tests/functional/afr/heal/test_self_heal_daemon_process.py # TC is checking for conditions that doesn't happen in glusterfs. It is failing in glusto too
     - tests/functional/afr/test_write_io_mount_point_resumed_quorum_restored.py # Probability of the TC passing is very less as the TC depends on the time of reboot, which varies
+    - tests/functional/snapshot/test_256_snapshots.py # Requires baremetal servers with big disks to create these many snapshots
```
Comment (srijan-sivakumar) [inline config/config.yml:69]: nit: Rather than saying big disk, we can state something like multiple disks and good amount of RAM.

**Review by srijan-sivakumar — CHANGES_REQUESTED** [review]

File: config/config.yml (hunk 1 above)
Comment (nik-redhat) [reply config/config.yml]: Done

**Review by srijan-sivakumar — APPROVED** [review]

**Pull request merged**